cypress/e2e/signUp.cy.js

```diff
@@ -1,12 +1,69 @@
-/// <reference types="cypress" />
+/// <reference types='cypress' />
 /// <reference types="../support" />
 
+const {generateNewUser} = require('../support/generateNewUser');
+
 describe('Sign Up page', () => {
-  before(() => {
+  beforeEach(() => {
```

Comment:
It's recommended to clear the database before each test to provide a clean state.